pkgs/development/nim-packages/frosty/default.nix

```diff
@@ -0,0 +1,8 @@
+{ fetchFromGitHub }:
+
+fetchFromGitHub {
```

Comment:
When I contributed Nitter to nixpkgs, I was not aware of the existence of fetch from Nimble. I believe it would be cleaner to fetch everything from Nimble in this case to ease the upgrade process of Nitter. I can do another pull request after this one to propose this change.

Reply:
I would leave `fetchFromGitHub`where it already is. `fetchNimble` is a hack, it fetches every dependency of the requested package and then throws them away. It should yield the same hash as `fetchFromGitHub`, so the two are interchangeable.